### ISSUE TITLE
Upload 1 image being able to crop 2 instances instantly...

### ISSUE THREAD

**Issue by b-creative** [sachatrauwaen/OpenContent#175]
I'm missing the possiblility to upload 1 image, crop it as a large image and in the next field call the same upload folder (without showing the uplad button) where this newly uploaded picture should reside, in order to crop a second smaller image. Now, I first must save, re-edit the module and then crop the smaller image using the image I uploaded before... Am I doing something wrong, is there a trick?

Thanks!!

**Comment by Timo-Breumelhof** [comment]
You are right, when you upload an image for a drop-down, the new folder content is not updated for the other drop-down(s).
I think a change to this behavior this has been considered a while ago, but I don't remember how complex the change was TBH.

**Comment by b-creative** [comment]
I worked around it by adding a panel field where I added the second image field

**Comment by Timo-Breumelhof** [comment]
@b-creative I talked about this with @sachatrauwaen is this still a needed requirement for you? @schotman

**Comment by b-creative** [comment]
I made these workarounds a couple of time now.... I can live without it, but it makes it easier when the workaround is not needed. Also, a panel suggests always being able to upload more then 1 item.... this is also not always ideal.
